modules/logging-release-notes-6-0-0.adoc

```diff
@@ -0,0 +1,121 @@
+
+[id="logging-release-notes-6-0-0-compatibility-support-matrix"]
+= Compatibility and support matrix
+
+In the table, components are marked with the following statuses:
+
+[horizontal]
+TP:: Technology Preview
+GA:: General Availability
+
+The components or features in link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] status are experimental and are not intended for production use.
+
+[NOTE]
+====
+To know about compatible {product-title} versions for {logging-uc} releases, see link:https://access.redhat.com/product-life-cycles?product=Red%20Hat%20OpenShift%20Logging[Product Life Cycles].
+====
+
+.Components support matrix
+[options="header"]
+|===
+
+| {logging-uc} Version 6+| Component Version
+
+| Operator | `eventrouter` | `logfilemetricexplorer` | `loki` | `lokistack-gateway` | `opa-openshift` | `vector`
+
+|6.0 | 0.4 (GA) | v1.1 (GA) | v3.1.0 (GA) | v0.1 (GA) | v0.1 (GA) | v0.37.x (GA)
+
+|===
+
+[id="logging-release-notes-6-0-0-removal-notice"]
+== Removal notice
+
+In Logging 6.0, the support is removed for:
+
+* Red Hat managed Elasticsearch log storage solution. For example, elasticsearch-operator
+* Red Hat managed log visualization using Kibana and Console Log View Plugin
+* Fluentd-based log collector implementations
++
+[NOTE]
+====
+The log collector deployments based on Fluentd will migrate to Vector
+====
+* ClusterLogging.logging.openshift.io custom resource
+* ClusterLogForwarder.logging.openshift.io custom resource
+
+[NOTE]
+====
+In order to continue to use Elasticsearch and Kibana managed by the elasticsearch-operator, the administrator must modify those object's ownerRefs before deleting the ClusterLogging resource.
+====
+
+[id="logging-release-notes-6-0-0-enhancements"]
+== New features and enhancements
+=== Log Collection
+
+* Before this update, the Vector collector deployments had no limits on memory and CPU usage. With this update, default requests and limits are set based on the recommendations from the Vector documentation. (link:https://issues.redhat.com/browse/LOG-4745[LOG-4745])
+
+* This enhancement brings Vector in line with the upstream Vector version v0.37.1. (link:https://issues.redhat.com/browse/LOG-5296[LOG-5296])
+
+* With this release, an alert triggers when log collectors buffer logs to a node's file system and use over 15% of the available space. This may indicate the log collectors are facing back pressure from their outputs, and administrators should act to prevent potential issues on the cluster node. (link:https://issues.redhat.com/browse/LOG-5381[LOG-5381])
+
+* This enhancement updates the selectors for all components to use common Kubernetes labels. (link:https://issues.redhat.com/browse/LOG-5906[LOG-5906])
+
+* With this update, the collector configuration now deploys as a ConfigMap instead of a secret. This change allows users to view and edit the configuration when the collector is not managed. (link:https://issues.redhat.com/browse/LOG-5599[LOG-5599])
```

Comment:
```suggestion
* With this update, the collector configuration will be deployed as a ConfigMap instead of a Secret. This change allows users to view and edit the configuration when the ClusterLogForwarder is set to Unmanaged. (link:https://issues.redhat.com/browse/LOG-5599[LOG-5599])
```